src/pl/wkos/textstats/TextStats.java

```diff
@@ -0,0 +1,60 @@
+package pl.wkos.textstats;
+
+public class TextStats {
```

Comment:
Jak dobrze widzę to to jest tak zwana klasa `Utils` wtedy potrzebny jest bezparametrowy finalny konstruktor oraz klasa powinna być finalna. Wspomnę dzisiaj o tym na lekcji

Reply:
Poczekam do komentarza na lekcji.